src/set_theory/cardinal.lean

```diff
@@ -609,6 +609,13 @@ theorem sum_le_sum {ι} (f g : ι → cardinal) (H : ∀ i, f i ≤ g i) : sum f
 ⟨(embedding.refl _).sigma_map $ λ i, classical.choice $
   by have := H i; rwa [← quot.out_eq (f i), ← quot.out_eq (g i)] at this⟩
 
+lemma mk_le_mk_mul_of_mk_preimage_le (c : cardinal) (f : α → β) (hf : ∀ b : β, #(f ⁻¹' {b}) ≤ c) :
```

Comment:
```suggestion
lemma mk_le_mk_mul_of_mk_preimage_le {c : cardinal} {f : α → β} (hf : ∀ b : β, #(f ⁻¹' {b}) ≤ c) :
```

Reply:
I'm happy to make `c` implicit since it appears in the conclusion, but I'm not so sure about `f`. If you want to invoke this lemma without providing the proof of `hf` right away, then keeping `f` explicit would be helpful.

Reply:
Does the style guide say anything about this? Otherwise, I guess it's a matter of personal preference.

Reply:
I would rather err on the side of keeping things explicit.